packages/components/package.json

```diff
@@ -96,7 +96,7 @@
     "react-leaflet": "^2.6.1",
     "react-leaflet-cluster": "1.0.3",
     "react-leaflet-geoman-v2": "^0.2.0",
-    "react-leaflet-v4": "npm:react-leaflet@4.1.0",
+    "react-leaflet-v3": "npm:react-leaflet@3.1.0",
```

Comment:
@lilitkarapetyan - Is there a specific reason/dependency for dropping react-leaflet back to v3?

Reply:
The GitHub repo suggests v4 is now the pre-requisite:
![image](https://user-images.githubusercontent.com/1108513/209670008-23efa4f9-98c8-491c-be87-6c53bcc5f6a9.png)

Reply:
Hello @IanMayo - The `"react-leaflet-cluster"` in  `"react": "^16.12.0"`  version is not working. We need to decrease `“react-leaflet": 3.1.0"` , to be able to work with `react-leaflet-cluster` and so that is why I decrease `react-leaflet ` version `3.1.0` to be able to work with `react-leaflet-cluster` in `react 16.12.0` version.

Reply:
Or we can try to increase the react version 18+

Reply:
I think I need to update `react version` to `18+ ` to be able to work with version` “react-leaflet-cluster: 4.0.1”` without problems.

Reply:
Hey, I was just thinking: "is now the right time to update to React 18?"

Reply:
> I think I need to update `react version` to `18+ ` to be able to work with version` “react-leaflet-cluster: 4.0.1”` without problems.

Yes, let's try updating to 18, please.

Do you think we should update in a fresh branch off `develop`?

Reply:
Okay, I will update react version to 18+

Reply:
I think yes